### DISCUSSION TITLE
Introducing myself and applying for maintainer role

### DISCUSSION THREAD

**Discussion by Snailedlt** [devicons/devicon#1263] [General]
Hi guys! I'd like to apply for a maintainer role here at devicons.
I already posted this in the Discord server, but since this community is more active here on GitHub, I'll repost it here too :)

## ABOUT ME

I'm Jørgen K. Hagen, from Norway. Most of you know me as Snailedlt on both Discord and GitHub.I am a junior dev/consultant with just over a year of work experience. I have experience with a lot of different fields in IT, here are some: Fullstack web dev, Fullstack android dev, ETL, CI/CD & workflow + testing automation, Discord bot dev. I am currently working in a CI/CD role, and I'm an admin/maintainer for the open source project Splitscreen.me https://github.com/SplitScreen-Me

## ME AND DEVICONS

I joined this community because I wanted to add some SVG icons of my skills on my profile README, and this was the perfect tool. I then started to contribute some icons, documentation and features to the devicons project.
My devicons work so far:
- New icon: SQLite
https://github.com/devicons/devicon/pull/926
- New icon: Unreal Engine
https://github.com/devicons/devicon/pull/939
- Updated CONTRIBUTING.md with devicon line icons
https://github.com/devicons/devicon/pull/941
- Added recommended tools and resources to README.md
https://github.com/devicons/devicon/pull/975
- Added altnames for all icons
https://github.com/devicons/devicon/pull/1167

Other than this I have also been active in reviewing PR's lately, since I find I can do that even when I have little time, and when I'm not on my PC.

## WHY I WANT TO JOIN

I would like to join as a maintainer, and help discuss and make important changes to make devicons more feature rich, and even more automated. I'll also help with PR reviews, and will look forward to continue learning from you guys's work. Check out my GitHub and Linkedin profiles below to know which technologies and projects I've worked with in the past. As well as my Pluralsight account to see which courses I've taken.

## MY SOCIALS

GitHub: https://github.com/Snailedlt
Linkedin: https://www.linkedin.com/in/jorgenkh
Pluralsight: https://app.pluralsight.com/profile/jorgenh

**PS:**
I also have experience as a Discord mod and Discord bot developer in a medium/large sized server (7.5k members), so I can help with that too.

**Comment by amacado** [comment]
I appreciate your work in this project and approve your application! ❤️ @Panquesito7?

**Comment by Panquesito7** [comment]
Amazing! Thanks for contributing to Devicon 🎉 I accept and would like you to join the team!
